resources/metadata/abapEnvironmentCheckoutBranch.yaml

```diff
@@ -52,7 +52,7 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
-        mandatory: true
+        mandatory: false
```

Comment:
```suggestion
```

Reply:
has been removed consistently for all false values